Boolean.CSharp.Main/Manager.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection.Metadata;
+using System.Runtime.CompilerServices;
+using System.Text;
+using System.Transactions;
+
+
+namespace Boolean.CSharp.Main
+{
+    public class Manager 
+    {
+        private static List<Customer> _users= new List<Customer>();
+
+        public static List<Customer> Users { get { return _users;  } }
+        public Manager()
+        {
+        }
+
+        public void registerCustomer(Customer user)
+        {
+            _users.Add(user);
+        }
+
+        public static void acceptOverdraft(GeneralAccount account, float amount)
```

Comment:
overdraft is different, it is a single value indiciating how much below 0 the account can go.